tests/schemas/api_protocol.json

```diff
@@ -317,161 +317,7 @@
-                                        "submit_payload_signature": {
-                                            "allow_none": false,
-                                            "required": true,
-                                            "type": "bytesField"
-                                        },
-                                        "submitted_on": {
-                                            "allow_none": false,
-                                            "required": true,
-                                            "type": "DateTime"
-                                        },
-                                        "submitter_der_x509_certificate": {
-                                            "allow_none": false,
-                                            "required": true,
-                                            "type": "bytesField"
-                                        }
-                                    }
-                                },
-                                "type": "Nested"
-                            },
-                            "required": true,
-                            "type": "List"
-                        },
-                        "status": {
-                            "allow_none": false,
-                            "required": true,
-                            "type": "CheckedConstant",
-                            "value": "ok"
-                        }
-                    },
-                    "serializing": "msgpack"
-                },
-                "req": {
-                    "fields": {
-                        "cmd": {
-                            "allow_none": false,
-                            "required": true,
-                            "type": "String"
-                        }
-                    },
-                    "serializing": "msgpack"
-                }
-            },
-            "pki_enrollment_reject": {
-                "rep": {
-                    "fields": {
-                        "status": {
-                            "allow_none": false,
-                            "required": true,
-                            "type": "CheckedConstant",
-                            "value": "ok"
-                        }
-                    },
-                    "serializing": "msgpack"
-                },
-                "req": {
-                    "fields": {
-                        "cmd": {
-                            "allow_none": false,
-                            "required": true,
-                            "type": "String"
-                        },
-                        "enrollment_id": {
-                            "allow_none": false,
-                            "required": true,
-                            "type": "UUID"
-                        }
-                    },
-                    "serializing": "msgpack"
-                }
-            }
-        },
+        "AUTHENTICATED": {},
```

Comment:
Maybe you can remove this line too now

Reply:
I left this line on purpose because `test_api_protocol_compat` seems to require it to check the protocol compat.

Reply:
Yes but you can modify the requirement since there are no `authenticated_cmds` left

Reply:
I modified how requirements are generated to skip empty categories. This line can now be removed.